### ISSUE TITLE
Plans / Wishlist

### ISSUE THREAD

**Issue by Samutz** [Samutz/sm64-port-android#1]
List of what I want to do with this. It will likely be slow going while I'm the only one working on them and I'm not super experienced with the languages used in this port.

- [ ] Rework touch controls:
  - [x] Automatic touch screen: #5
  - [x] Add L button for cheats: #6
  - [ ] Replace C-buttons with right analog control when analog camera is enabled
  - [ ] Change skin or add optional button skins / add transparency option (personally I don't like the default pixelated buttons)
  - [ ] Fix the touchscreen sending unwanted B button inputs on some devices
- [ ] Allow toggling these features either in-game or in a launcher:
  - [ ] 60 FPS / 30 FPS
  - [ ] Touch screen buttons
  - [ ] No draw distance
    - [ ] Allow setting custom distance
  - [ ] DynOS (for when things go wrong with mods)
  - [ ] External Data (for when things go wrong with mods)
- [ ] Move assets from rom to a separate file, such as an OBB file. I would like to be able to distribute an APK that contains no copyrighted material. Then the user would only need to build an OBB file from their rom.

Basically I want to make this port more accessible.

**Comment by Samutz** [comment]
https://samutz.com/docs/books/video-games/page/building-sm64ex-on-android
